xamarin/Xamarin.Forms
# Allow to create more easily SwipeItems

### Description of Change ###

Added new constructors to SwipeItems to allow to create more easily using C#. 
This allows to create SwipeItems as follows:
```
var swipeView = new SwipeView
{
	LeftItems = new SwipeItems(deleteSwipeItem)
	{
		Mode = SwipeMode.Reveal
	}
};
```

### Issues Resolved ### 

- fixes #8753

### API Changes ###
 
 None

### Platforms Affected ### 

- Core

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the Issue8753. If you can see and open the SwipeView, the test has passed.

### PR Checklist ###
